.github/workflows/test-package.yml

```diff
@@ -157,6 +169,52 @@ jobs:
           dart run ffigen --config ffigen.yaml
           diff lib/src/third_party/jni_bindings_generated.dart __old_bindings
 
+  ## Run tests for package:jni on windows, just to confirm it works.
+  ## Do not, for example, collect coverage or check formatting.
+  test_jni_windows_minimal:
+    needs: [analyze_jni]
+    runs-on: windows-latest
+    defaults:
+      run:
+        working-directory: ./jni
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          cache: true
+          cache-key: 'flutter-:os:-:channel:-:version:-:arch:-:hash:'
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - run: Add-Content $env:GITHUB_PATH "$env:JAVA_HOME\bin\server"
+      - run: dart pub get
+      - run: dart run jni:setup
+      - run: dart test -j 1
```

Comment:
Why does it fail without `-j 1`? Is our API not concurrency-safe?

Can we try to figure out what concurrency guarantees we can and cannot make? (Also, why doesn't it fail on Linux?)

Reply:
The creation and deletion of some stub files conflicts obviously. Windows seems just more brittle so errors were caught.

It can be countered by attaching timestamp to every temp file. But is that worth the effort?

Reply:
> some stub files conflicts

Is that files that we create? Or are these created by `dart run`? I'd like to understand the issue.

Reply:
Yes for example __temp_pom.xml.

Reply:
> It can be countered by attaching timestamp to every temp file. 

I think it is more common to use https://api.dart.dev/stable/2.18.1/dart-io/Directory/createTemp.html.

And making tests conflict with each other is a bug in our tests that we should fix. (Feel free to fix either in this PR or to file an issue and make a separate PR.)